cmd/lncli/commands_test.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseChannelPointStr(t *testing.T) {
+	testCases := []struct {
+		channelPoinStr    string
+		channelPointIsNil bool
+		outputIndex       uint32
+		err               error
+	}{
```

Comment:
Nice tests! If you want, you can get rid of one level of indentation by using the following syntax:

```
testCases := []{
...
}{{
...
}, {
...
}}
```